framework/log/Target.php

```diff
@@ -60,6 +60,12 @@
      */
     public $logVars = ['_GET', '_POST', '_FILES', '_COOKIE', '_SESSION', '_SERVER'];
     /**
+     * @var array list of keys which will be excluded from [[logVars]]
+     * Default to []
+     * @see lofVars
```

Comment:
I fexed this. https://github.com/yiisoft/yii2/pull/11462/commits/7bc5ae4881e7cea80285daf28fea9cab09db8142